### DISCUSSION TITLE
씨줄과 날줄 | kciter.so

### DISCUSSION THREAD

**Discussion by giscus** [kciter/kciter.github.io#31] [Announcements]
# 씨줄과 날줄 | kciter.so

아무것도 몰라도 타입 검사를 그냥 '사용'할 수는 있다. 하지만 타입 검사를 제대로 '활용'하는 것은 전혀 다른 문제다. 아무래도 타입은 프로그래밍을 할 때 가장 많이 고려하는 요소라고 할 수 있다. 우리는 프로그래밍 언어가 제공하는 문법을…

https://kciter.so/posts/warp-and-weft/

**Comment by cadenzah** [comment]
본문의 Type-Driven Development 글 링크가 잘못되어있습니다. /posts 가 주소에 빠진 듯해요.

**Reply by kciter** [reply]
수정했습니다. 감사합니다! 😄
